spark/v3.0/spark/src/main/java/org/apache/iceberg/spark/actions/BaseDeleteOrphanFilesSparkAction.java

```diff
@@ -186,6 +202,27 @@ private DeleteOrphanFiles.Result doExecute() {
     return new BaseDeleteOrphanFilesActionResult(orphanFiles);
   }
 
+  static Dataset<String> getOrphanFilesDF(Dataset<Row> actualFileDF,
```

Comment:
We usually try to avoid using `get` as it has almost to meaning. What about `findOrphanFiles` and returning a list of strings?